mergify_engine/queue/merge_train.py

```diff
@@ -155,15 +156,20 @@ def deserialize(cls, train: "Train", data: "TrainCar.Serialized") -> "TrainCar":
             data["state"] = "created"
         return cls(train, **data)
 
-    def _get_embarked_refs(self, include_my_self=True):
+    def _get_embarked_refs(self, include_my_self=True, markdown=False):
         pull_refs = ", ".join(
             [
                 f"#{p}"
                 for p in self.parent_pull_request_numbers
                 + ([self.user_pull_request_number] if include_my_self else [])
             ]
         )
```

Comment:
Can you reformat this as:
`foobar, #123 and #456`

using  `and` rather than a comma between the before-last and last elements. That sounds more english.